server/services/AI/AI.js

```diff
@@ -0,0 +1,58 @@
+const fs = require('fs');
+
+/* Random card picked*/
+exports.autoPickCard = cards => {
```

Comment:
Maybe rename this to `pickRandomCard`? It can also be reused when you are doing play random card in GameLogic